packages/zenn-cli/src/client/components/Sidebar.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import styled from 'styled-components';
 
 // icons
-import ArrowBackOutlinedIcon from '@material-ui/icons/ArrowBackOutlined';
-import ArrowForwardOutlinedIcon from '@material-ui/icons/ArrowForwardOutlined';
-import SortRoundedIcon from '@material-ui/icons/SortRounded';
+import { MdOutlineArrowBack } from 'react-icons/md';
+import { MdOutlineArrowForward } from 'react-icons/md';
+import { MdSort } from 'react-icons/md';
```

Comment:
```suggestion
import { 
  MdSort,
  MdOutlineArrowBack
  MdOutlineArrowForward,
 } from 'react-icons/md';
```

のようにまとめられるので、まとめた方がいいかもですね。